src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -382,6 +384,12 @@ def _filter_file(tarinfo):
                     ".dockerignore: skip checking '{}'".format(docker_file_path))
                 return tarinfo
 
+            # ignore common vcs dir or file
+            if tarinfo.name in common_vcs_ignore_list:
```

Comment:
does this approach work for directories like `.git`? I thought this func was invoked recursively on all files, rather than the directories as well? I.e., `.gitignore`, `.git` would be ignored, but what about all the objects contained within them?

Reply:
If .git folder is ignored, the files in .git folder will be ignored.

Reply:
so, just to clarify: for a folder structure:

```
.git/
      foo
      bar
```

this method should get called with:

`.git`
`.git/foo`
`.git/bar`

and by only specifying `.git`, the `.git/foo` and `.git/bar` do not appear? I find that very peculiar behavior if so but believe you.